src/com/tkjngine/gametable/monsterwounds/PogMenuRenderListener.java

```diff
@@ -0,0 +1,117 @@
+                    if (null == pogRegistry.get(activePog))
+                    {
+                        MonsterData monsterData = new MonsterData();
+                        PogAdapter pogAdapter = new PogAdapter(monsterData, activePog);
+                        pogRegistry.put(activePog, pogAdapter);
+                    }
+                }
+            });
+
+            menu.addSeparator();
+            menu.add(item);
+        } else {
+            final MonsterData monsterData = pogAdapter.getMonsterData();
+            final JMenu monsterWoundsMenu = new JMenu("Monster Wounds");
+            menu.add(monsterWoundsMenu);
+            JMenuItem item = null;
+
+            item = new JMenuItem("-1 Wound");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    monsterData.decWounds();
+                }
+            });
+            monsterWoundsMenu.addSeparator();
+            monsterWoundsMenu.add(item);
+
+            item = new JMenuItem("+1 Wound");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    monsterData.incWounds();
+                }
+            });
+            monsterWoundsMenu.addSeparator();
+            monsterWoundsMenu.add(item);
+
+            item = new JMenuItem("Refill Wounds");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    monsterData.setWounds(monsterData.getMaxWounds());
+                }
+            });
+            monsterWoundsMenu.addSeparator();
+            monsterWoundsMenu.add(item);
+
+            item = new JMenuItem("Set Wounds");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    // max wounds dialog
+                    WoundsDialogue dialogue = new WoundsDialogue(monsterData.getWounds(), monsterData.getMaxWounds());
+                    dialogue.setLocationRelativeTo(GametableFrame.getGametableFrame().getGametableCanvas());
+                    dialogue.setVisible(true);
+                    monsterData.setWounds(dialogue.getValue());
+                }
+            });
+            monsterWoundsMenu.addSeparator();
+            monsterWoundsMenu.add(item);
+
+            item = new JMenuItem("Set Max Wounds");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    // max wounds dialog
+                    MaxWoundsDialogue dialogue = new MaxWoundsDialogue(monsterData.getMaxWounds());
+                    dialogue.setLocationRelativeTo(GametableFrame.getGametableFrame().getGametableCanvas());
+                    dialogue.setVisible(true);
+                    monsterData.setMaxWounds(dialogue.getValue());
```

Comment:
From a usability perspective, I wonder if it's worth also setting current wounds to the value too. I can't think of any situation where you would want to set max wounds, and not set current wounds to match. The only occasion you would set max wounds is when you create the pog, and by default it would have full health.